tools/topology/topology2/platform/intel/dmic-generic.conf

```diff
@@ -1,5 +1,5 @@
 Define {
-       DMIC0_PCM_NAME	"DMIC"
+       DMIC0_PCM_NAME	"DMIC Raw"
```

Comment:
I guess I'm very late to the came, but given there is gain and EQ, calling this "Raw" is bit of a stretch. Not blocking on this, if the naming has been agreed.